src/routes/Quote.jsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable max-len */
+import './Quote.css';
```

Comment:
- [ ] Dear @dennis-every, It's not recommended to disable `linters` in our code base, Linters are here to make sure our code is clean and according to the standards. Pleae, remove disabled linter rules.

Reply:
Hi @TadesseDev, ok, tks for the feedback!